.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This is extra code.. I don't think this belongs with the game, you may need to add it to .gitignore or something. Look it up on google I guess.

Reply:
I think you still need to address this issue.